Radius(x).py

```diff
@@ -6,31 +6,68 @@
 @author: amelielaurens
```

Comment:
I am not sure what is the best way to do it, but we need more descriptions to details the code.
So for now I would put in the description of each script:
- bibliographic data of the paper from which you are implementing this model
- a short description of the purpose of the code and its output
- key things to do before running the script
- mention that all data are in Si units.

Reply:
All of these can be automatized using docstrings (please see comment above), if docstrings are properly written, python can generate a documentation out of them using pdoc (https://pdoc3.github.io/pdoc/), powerful stuff.